extensions/amp-story-auto-ads/0.1/_locales/index.js

```diff
@@ -0,0 +1,159 @@
+import LocalizedStringmk from './mk.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringml from './ml.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringmn from './mn.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringmr from './mr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringms from './ms.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringmy from './my.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringne from './ne.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringnl from './nl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringno from './no.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringpa from './pa.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringpl from './pl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringptBR from './pt-BR.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringptPT from './pt-PT.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringro from './ro.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringru from './ru.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsi from './si.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsk from './sk.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsl from './sl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsq from './sq.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsr from './sr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsv from './sv.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsw from './sw.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringta from './ta.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringte from './te.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringth from './th.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringtr from './tr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringuk from './uk.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringur from './ur.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringvi from './vi.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzhCN from './zh-CN.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzhTW from './zh-TW.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzu from './zu.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+
+export default [
```

Comment:
For the purposes of this PR we just want to import all as an array to test them. 
We can still import them individually by doing e.g. `import LocalizedStringzu from './zu.json' assert {type: 'json'};` which I think is actually safer, because the build system will complain if `./zu.json` cannot be found right away.